docs/environment.yml

```
@@ -29,4 +29,5 @@ dependencies:
     - nitime
     - nilearn
     - niworkflows>=0.5.4,<0.6
+    - smriprep>=0.0.4,<1.0.0
```

I think this will ensure that we get everything in the 0.0.x series and no pre-releases of 0.1.x or later:

```suggestion
    - smriprep>=0.0.4,<0.1.0a0
```

Or are you thinking sMRIPrep 1.0 is coming very soon?

---

I was actually thinking whether sMRIPrep should keep up versions with fMRIPrep - when we are sure about this, push both to 1.3.0.